requirements-test.txt

```
@@ -3,4 +3,4 @@ attrs>=17.4.0
 pytest-django>=3.4.2,<3.10.0
 pytest-asyncio>=0.10.0,<0.15.0
 pytest-cov>=2.7.0,<2.11.0
-openwisp-utils[qa]~=0.6.0
+openwisp-utils[qa]>=0.6.0,<=0.7.0
```

```suggestion
openwisp-utils[qa]>=0.6.0,<0.8
```
This way will support all minor versions of openwisp-utils 0.7